test/e2e/View.test.ts

```diff
@@ -122,12 +125,23 @@ describe('BPMN Visu JS', () => {
     await expect(page.title()).resolves.toMatch('BPMN Visu JS');
   });
 
+  function expectElementOfType(cellId: string, shapeKind: ShapeBpmnElementKind): void {
```

Comment:
:question: what do think about renaming the function into `expectModelHasCell` or `expectModelContainsCell`?
I would like to emphasize that we are expecting elements in the mxgraph model and not in the DOM

Reply:
The 2 names that you propose are good to me :)

Reply:
Ok, so I am going to commit a change for `expectModelContainsCell`